example_byteslice_test.go

```diff
@@ -11,7 +11,25 @@ func ExampleReverse() {
 
 func ExampleLPad() {
 	data := []byte{0x55, 0xDA, 0xBA}
-	
+
 	fmt.Printf("%x\n", LPad(data, 5, 0x22))
 	// Output: 222255daba
 }
+
+func ExampleToggle() {
+	data := []byte("bada55")
+	toggle := []byte("101011")
+
+	output, _ := Toggle(data, toggle)
+	fmt.Printf("%x\n", output)
+	// Output: 535155510404
```

Comment:
The output need to be `bada55` here, not a byte slice of 6 bytes, so the **data** and **toogle** need to be both a byte slice of 3 bytes.
`[]byte("bada55")` are not the same as `[]byte(0xba, 0xda, 0x55)`

Reply:
I've changed the slice to not use a string, I've also added two examples, one simple, one regular one. I've also squashed the commits down to one so as not to clutter commit history